src/commands/instances/player/PlayerSetFlagCommand.ts

```diff
@@ -43,8 +44,13 @@ class PlayerSetFlagCommand implements Command {
     }
 
     try {
-      const result = await updateCountry(username, countryCode);
-      response.message = result.message;
+      await updateCountry(username, countryCode);
+
+      response.message = `${message.user} changed \`${username}\`'s country to ${
+        Object.entries(CountryProps)
```

Comment:
if you only want the country's name, you could do `CountryProps[countryCode].name` instead of these finds

the idea behind those Props objects is to have O(1) access to those props (MetricProps, PeriodProps, CountryProps, etc)